### PR TITLE
Switch to using structured logging

### PR DESCRIPTION
**Pull request by nathanielc**
Now using go.uber.org/zap logging package for structured logging. This follows the same structure as InfluxDB

- [x] Rebased/mergable
- [x] Tests pass
- [ ] CHANGELOG.md updated

### CODE REVIEW

**Comment by desa** [comment]
Just wanted to note here that there's going to be some breaking changes coming to the `go.uber.org/zap`

https://github.com/influxdata/influxdb/issues/7770

**Comment by rossmcdonald** [comment]
On startup, I'm seeing the first two messages in the incorrect format:
```
2017/02/03 11:12:06 Using configuration at: /Users/ross/.kapacitor/config
2017/02/03 11:12:06 setting log level INFO
ts=2017-02-03T17:12:06Z level=info msg="Kapacitor starting" version= branch=unknown commit=unknown go-version=go1.7.3
```

Not sure if that's expected.

File: services/httpd/handler.go [hunk 1]
```diff
@@ -81,14 +81,16 @@ type Handler struct {
...
+	logger zap.Logger
 	// Detailed logging of write path
 	// Uses normal logger
 	writeTrace bool
 
 	// Common log format logger.
-	// This logger does not use log levels with wlog.
+	// This logger does not use log levels.
 	// Its simply a binary on off from the config.
 	clfLogger *log.Logger
```
Comment (desa) [inline services/httpd/handler.go]: Is this still intended to be used?

Comment (nathanielc) [inline services/httpd/handler.go]: Yes, for now anyways. But I would like to more off of it at some point so that all logs have the same format.

**Comment by nathanielc** [comment]
@desa Thanks for the heads up on influxdata/influxdb#7770, I think I'll hold off merging anything till that lands.

**Pull request closed without merging**